projects/1.18/assets/macaws-roofs-oh-the-biomes-youll-go/macawsroofsbyg/lang/zh_cn.json

```
@@ -0,0 +1,452 @@
+	"block.macawsroofsbyg.rainbow_eucalyptus_top_roof":"彩虹桉木屋顶顶端",
+	"block.macawsroofsbyg.rainbow_eucalyptus_lower_roof":"彩虹桉木平缓屋顶底端",
+	"block.macawsroofsbyg.rainbow_eucalyptus_steep_roof":"彩虹桉木陡峭屋顶",
+	"block.macawsroofsbyg.rainbow_eucalyptus_upper_lower_roof":"彩虹桉木顶端平缓屋顶",
+	"block.macawsroofsbyg.rainbow_eucalyptus_upper_steep_roof":"彩虹桉木顶端陡峭屋顶底端",
+	"block.macawsroofsbyg.rainbow_eucalyptus_planks_roof":"彩虹桉木木板屋顶",
+	"block.macawsroofsbyg.rainbow_eucalyptus_planks_attic_roof":"彩虹桉木木板天窗屋顶",
+	"block.macawsroofsbyg.rainbow_eucalyptus_planks_cube_roof":"彩虹桉木木板方块屋顶",
+	"block.macawsroofsbyg.rainbow_eucalyptus_planks_top_roof":"彩虹桉木木板屋顶顶端",
+	"block.macawsroofsbyg.rainbow_eucalyptus_planks_lower_roof":"彩虹桉木木板平缓屋顶",
+	"block.macawsroofsbyg.rainbow_eucalyptus_planks_steep_roof":"彩虹桉木木板陡峭屋顶底端",
+	"block.macawsroofsbyg.rainbow_eucalyptus_planks_upper_lower_roof":"彩虹桉木木板平缓屋顶顶端",
+	"block.macawsroofsbyg.rainbow_eucalyptus_planks_upper_steep_roof":"彩虹桉木木板陡峭屋顶顶端",
+	"block.macawsroofsbyg.redwood_roof":"红杉木屋顶",
+	"block.macawsroofsbyg.redwood_attic_roof":"红杉木天窗屋顶",
+	"block.macawsroofsbyg.redwood_cube_roof":"红杉木方块屋顶",
+	"block.macawsroofsbyg.redwood_top_roof":"红杉木屋顶顶端",
+	"block.macawsroofsbyg.redwood_lower_roof":"红杉木平缓屋顶底端",
+	"block.macawsroofsbyg.redwood_steep_roof":"红杉木陡峭屋顶",
+	"block.macawsroofsbyg.redwood_upper_lower_roof":"红杉木顶端平缓屋顶",
+	"block.macawsroofsbyg.redwood_upper_steep_roof":"红杉木顶端陡峭屋顶底端",
+	"block.macawsroofsbyg.redwood_planks_roof":"红杉木木板屋顶",
+	"block.macawsroofsbyg.redwood_planks_attic_roof":"红杉木木板天窗屋顶",
+	"block.macawsroofsbyg.redwood_planks_cube_roof":"红杉木木板方块屋顶",
+	"block.macawsroofsbyg.redwood_planks_top_roof":"红杉木木板屋顶顶端",
+	"block.macawsroofsbyg.redwood_planks_lower_roof":"红杉木木板平缓屋顶",
+	"block.macawsroofsbyg.redwood_planks_steep_roof":"红杉木木板陡峭屋顶底端",
+	"block.macawsroofsbyg.redwood_planks_upper_lower_roof":"红杉木木板平缓屋顶顶端",
+	"block.macawsroofsbyg.redwood_planks_upper_steep_roof":"红杉木木板陡峭屋顶顶端",
+	"block.macawsroofsbyg.skyris_roof":"天际木屋顶",
+	"block.macawsroofsbyg.skyris_attic_roof":"天际木天窗屋顶",
+	"block.macawsroofsbyg.skyris_cube_roof":"天际木方块屋顶",
+	"block.macawsroofsbyg.skyris_top_roof":"天际木屋顶顶端",
+	"block.macawsroofsbyg.skyris_lower_roof":"天际木平缓屋顶底端",
+	"block.macawsroofsbyg.skyris_steep_roof":"天际木陡峭屋顶",
+	"block.macawsroofsbyg.skyris_upper_lower_roof":"天际木顶端平缓屋顶",
+	"block.macawsroofsbyg.skyris_upper_steep_roof":"天际木顶端陡峭屋顶底端",
+	"block.macawsroofsbyg.skyris_planks_roof":"天际木木板屋顶",
+	"block.macawsroofsbyg.skyris_planks_attic_roof":"天际木木板天窗屋顶",
+	"block.macawsroofsbyg.skyris_planks_cube_roof":"天际木木板方块屋顶",
+	"block.macawsroofsbyg.skyris_planks_top_roof":"天际木木板屋顶顶端",
+	"block.macawsroofsbyg.skyris_planks_lower_roof":"天际木木板平缓屋顶",
+	"block.macawsroofsbyg.skyris_planks_steep_roof":"天际木木板陡峭屋顶底端",
+	"block.macawsroofsbyg.skyris_planks_upper_lower_roof":"天际木木板平缓屋顶顶端",
+	"block.macawsroofsbyg.skyris_planks_upper_steep_roof":"天际木木板陡峭屋顶顶端",
+	"block.macawsroofsbyg.sythian_roof":"斯基泰菌柄屋顶",

```

```suggestion
	"block.macawsroofsbyg.sythian_roof":"斯基泰木屋顶",
```
下同，这些就不用管原文了（个人意见）
和其他木制品名称统一